File: morning_exercises/day3.cpp
#include <cmath>
#include <iostream>
#include <ostream>
// #include <vector>

bool isFib(int n) {
    if (n < 0) return false;

    int root1 = (int)std::sqrt((5*n*n)+4);
    int root2 = (int)std::sqrt((5*n*n)-4);

    if (((5*n*n)+4)==root1*root1 || (((5*n*n)-4)==root2*root2)) return true;
    else return false;
}

void fibBuzz(int n) {
    for (int i=1;i<=n;i++) {
        if (isFib(i)==true && i%7!=0) std::cout << "Fib" << std::endl;
        else if (isFib(i)==true && i%7==0) std::cout << "FibBuzz" << std::endl;
        else if (isFib(i)==false && i%7==0) std::cout << "Buzz" << std::endl;
        else std::cout << i << std::endl;
    }
}

int main() {
    fibBuzz(10);
}

File: morning_exercises/day3.cs
public static class FibBuzz
{
    public static bool IsFib(int n)
    {
        if (n<0) return false;

        int root1 = (int)Math.Sqrt((5*n*n)+4);
        int root2 = (int)Math.Sqrt((5*n*n)-4);

        if ((root1*root1==((5*n*n)+4)) || (root2*root2==((5*n*n)-4))) return true;
        else return false;
    }

    public static void IsFibBuzz(int n)
    {
        for (int i=1;i<=n;i++) 
        {
            if (IsFib(i)==true && i%7!=0) Console.WriteLine("Fib");
            else if (IsFib(i)==true && i%7==0) Console.WriteLine("FibBuzz");
            else if (IsFib(i)==false && i%7==0) Console.WriteLine("Buzz");
            else Console.WriteLine(i);
        }
    }
}